allure-generator/src/main/java/io/qameta/allure/history/HistoryPlugin.java

```diff
@@ -37,7 +37,8 @@
 
     //@formatter:off
     private static final TypeReference<Map<String, HistoryData>> HISTORY_TYPE =
-        new TypeReference<Map<String, HistoryData>>() {};
+            new TypeReference<Map<String, HistoryData>>() {
```

Comment:
remove this change

PS you can configure IDEA: go to Editor -> Code Style -> Enable formatter markers in comments